spec/lib/transaction_history_search_spec.rb

```diff
@@ -44,6 +44,8 @@ def all_order_details_with_search
 
     # fake signing in as staff
     @controller.session_user = @staff
+    @controller.response = ActionDispatch::Response.new
+    @controller.request = ActionDispatch::Request.new('rack.input' => StringIO.new, content_type: 'html')
```

Comment:
When I added the `respond_to` in `TransactionSearch:: ClassMethods#define_search_method`   it causing the specs the fail with the error below. I traced the error into activesupport where it was calling request.negotiate_mime.

```
NoMethodError: undefined method `negotiate_mime' for nil:NilClass

Stack trace:
./lib/transaction_search.rb:44:in `block in define_search_method'
./spec/lib/transaction_history_search_spec.rb:146:in `block (4 levels) in <top (required)>'
```